app/metrics/support/counter_support.rb

```diff
@@ -7,6 +7,10 @@ def fetch_and_format(aggregate = nil)
     # The block on the deep_merge ensures that the final values
     # are added.  Without it, the value is the last value passed in.
     ordered_counters.each_with_object({}) do |values, result|
+      # Rails casts PG numeric as BD, which causes problems in the addition.
```

Comment:
How come is a problem? I'd have thought that floats would cause problems since they don't have precision